### ISSUE TITLE
Allow accid.ges on <accid>

### ISSUE THREAD

**Issue by ahankinson** [music-encoding/music-encoding#275]
Currently `@accid.ges` is not allowed on `<accid>`. From an e-mail exchange with Perry:

```
<accid> was intended to only operate in the visual domain, leaving info about
the performed accidental on  <note>.  But, I'm not sure this approach is valid.
When something that was an attribute gets promoted to object status, ideally
the properties associated with it come along as well.  The determination of whether
@accid.ges is a property of <accid> or of <note> is a significant issue.
```

This is related to #261, where it was decided that `@accid.ges` would refer to a performed, but not visual accidental. If we want to use `<accid>` to specify the accidental on a note, but have it represent a performed but not visible accidental, we will need to add `@accid.ges` to `<accid>`.

**Comment by zolaemil** [comment]
Interesting question. Do we not have the same problem with `<octave>`, `@oct` and `@oct.ges`?

**Comment by ahankinson** [comment]
I don't think so -- `<octave>` is not a child of `<note>` like `<accid>` is. So `@oct` information is stored on the note anyway, and the presence of `<octave>` is explicitly gestural and applied to a full passage. From: http://music-encoding.org/documentation/2.1.1/octave/ :

```
An indication that a passage should be performed one or more octaves above or below its written pitch.
```

**Comment by pe-ro** [comment]
Closed by [081eb58]
